internal/graph/check_test.go

```diff
@@ -2198,3 +2198,68 @@ func TestCycleDetection(t *testing.T) {
 		})
 	})
 }
+
+func TestTupleIDInSortedSet(t *testing.T) {
+	filter := func(tupleKey *openfgav1.TupleKey) (bool, error) {
+		if tupleKey.GetCondition().GetName() == "condition1" {
+			return true, nil
+		}
+		return false, fmt.Errorf("condition not found")
+	}
+
+	tests := []struct {
+		name          string
+		tuples        []*openfgav1.TupleKey
+		objectIDs     []string
+		expectedError bool
```

Comment:
**(suggestion):** Could we change this to a string? And match against the actual error message? Wdyt?

Reply:
Similar to https://github.com/openfga/openfga/pull/1840#discussion_r1716956867, the error message is generated from external function.  Hence, we don't want to tie the assertion to whatever error string the external function generate.